File: PIAO_FE/src/app/shared/models/classes/obbligo-legge-dto.ts
import { CampiTecniciDTO } from './campi-tecnici-dto';
import { DatiPubblicatiDTO } from './dati-pubblicati-dto';

export class ObbligoLeggeDTO extends CampiTecniciDTO {
  id?: number;
  idSezione23?: number;
  denominazione?: string;
  descrizione?: string;
  datiPubblicati?: DatiPubblicatiDTO[];
}

File: Common/Entity/src/main/java/it/ey/dto/ObbligoLeggeDTO.java
package it.ey.dto;

import lombok.*;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class ObbligoLeggeDTO extends CampiTecniciDTO{

    private Long id;

    private Long idSezione23;

    private String denominazione;

    private String descrizione;
    private List<DatiPubblicatiDTO> datiPubblicati;

}
